src/Bridge/Symfony/Bundle/Resources/config/doctrine_orm.xml

```diff
@@ -105,13 +105,13 @@
             <argument>%api_platform.collection.pagination.enabled_parameter_name%</argument>
             <argument>%api_platform.collection.pagination.items_per_page_parameter_name%</argument>
 
-            <tag name="api_platform.doctrine.orm.query_extension.collection" priority="-20" />
+            <tag name="api_platform.doctrine.orm.query_extension.collection" priority="16" />
```

Comment:
Or rather, `PaginationExtension` should be split into two. And there should be multiple passes / stages for the extensions.